src/main/java/mate/academy/bookstoreapp/service/impl/ShoppingCartServiceImpl.java

```diff
@@ -73,6 +86,25 @@ public void clearCart(Long userId) {
         shoppingCartRepository.save(shoppingCart);
     }
 
+    @Override
+    public void transformCartToOrder(Long id, CreateOrderRequestDto requestDto) {
+        Order order = orderMapper.toEntity(requestDto);

```

Comment:
create method createOrder in OrderServiceImpl and implement it there 
as I understood these logic related to the create order